pkg/operator/ceph/object/spec.go

```diff
@@ -418,6 +418,13 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) (v1.Container,
 		vaultVolMount := v1.VolumeMount{Name: rgwVaultVolumeName, MountPath: rgwVaultDirName}
 		container.VolumeMounts = append(container.VolumeMounts, vaultVolMount)
 	}
+
+	hostingOptions, err := c.addDNSNamesToRGWServer()
+	if hostingOptions != "" && err == nil {
+		container.Args = append(container.Args, hostingOptions)
+	} else if err != nil {
+		return v1.Container{}, err
+	}
```

Comment:
Please use the standard golang error handling conventions 

```suggestion
	if err != nil {
		return v1.Container{}, err 
	}
	if hostingOptions != "" {
		container.Args = append(container.Args, hostingOptions)
	}
```